scripts/linux/log_collection.sh

```diff
@@ -16,7 +16,7 @@ automate=false   # set to true if running via a JumpCloud command (recommended)
 # do not edit below
 #######
 
-version=1.0
+version=1.1
```

Comment:
```suggestion
version=1.0.1
```

Reply:
Let's start using semantic versioning on all these scripts. That's our standard elsewhere.